warehouse/accounts/views.py

```diff
@@ -53,6 +53,7 @@ def failed_logins(exc, request):
 @view_config(
     route_name="accounts.profile",
     renderer="accounts/profile.html",
+    uses_session=True,
```

Comment:
thanks @dstufft - can you please ping me on slack when you are around and we can go though this together.  I'm not sure I understand what I need to do here :)